README.md

```diff
@@ -0,0 +1,701 @@
+
+```json
+{
+  //...
+  "action": "document/update",
+  "conditions": { "document_ids": ["0A01.."] }
+}
+```
+
+The above capability gives authority to update only document `"0A01.."`.
+
+### `schema_ids`
+
+Including `schema_ids` in a capabilities conditions restricts the authority scope to only documents following the specified schemas.
+
+```json
+{
+  //...
+  "action": "document/update",
+  "conditions": { "schema_ids": ["events_0A01..", "resources_0B02.."] }
+}
+```
+
+The above capability gives authority to update any documents which are `"events"` or `"resources"`.
+
+### `to_timestamp`
+
+Including `to_timestamp` restricts a capabilities authority to only operations which contain a lower `timestamp` in their header.
+
+```json
+{
+  //...
+  "action": "document/update",
+  "conditions": {
+    "document_ids": ["0A01.."],
+    "to_timestamp": 1712226632
+  }
+}
+```
+
+The above capability restricts authority to document `"0A01"` and only operations containing a timestamp `<=` `1712226632`.
+
+At first glance it may seem that this (and the following `from_timestamp`) condition is duplicating behaviour achievable via a capabilities `expires` and `not_before` field. However on closer consideration we can see how these two fields can be configured to support network specific degrees of delay tolerance while still retaining recommended "least authority" practices. To demonstrate this we can use an example where a network delay of one day is tolerated. In this situation we can expect that after one day has passed all peers should have received all published operations for all documents they replicate (and have `document/read` capabilities for). For this network we might issue capabilities with the following values:
+
+```json
+{
+  //...
+  "expires": 1712310016, // one day later than to_timestamp
+  "action": "document/update",
+  "conditions": {
+    "document_ids": ["0A01.."],
+    "to_timestamp": 1712226632 // one day before expires
+  }
+}
+```
+
+By including an `"expires"` field which contains a timestamp one day ahead of the `"to_timestamp"` condition we are allowing operations within the capabilities scope to arrive up to one day late, but no longer. After `"expires"` has passed we can reject updates from authors depending on this capability, even if they would have fallen within the authorised scope. Correct configuration of this behaviour requires an understanding of the networks you will be using and the degree of delay you wish to tolerate. Misconfiguration has the potential to artificially introduce network partitions, where one peer accepted operations which another never received, however the capability used has now expired. This "deadlock" is expected to be resolved when new capabilities are issued, or if the operations are due for garbage collection in any case.
```

Comment:
use consistent language "one day later".

note that "ahead" is ambiguous here. Any time movement which uses a vague direction presumes a sense (direction) of the reader relative to time. e.g. Not everyone sees "i moved the event back" to mean th same thing !